### DISCUSSION TITLE
Product labels in the gallery

### DISCUSSION THREAD

**Discussion by MaxSouza** [breezefront/community#340] [2. Ideas]
Hello Everyone,

I'm making the Amasty Label module compatible with Breeze, and I'd like to suggest a change in the gallery code. The label is being replaced by the product image due to this property:

`vendor/swissup/module-breeze/view/frontend/web/js/components/gallery.js:54`
`this.image = this.stage.find('.stage img');`

The idea is to specify the image class instead of using the general image tag. Like:

`this.image = this.stage.find('.stage img.main-image');`

Before:

![image](https://github.com/user-attachments/assets/2685f208-476f-4c30-863f-c6c5b534200a)

After:

![image](https://github.com/user-attachments/assets/06b1f4d5-e2d0-43b8-8dec-b5f9858b314f)

That's a specific change, let me know if it is worth it

**Comment by vovayatsyuk** [comment]
Looks good to me! Just made the commit: https://github.com/breezefront/module-breeze/commit/07d567d17460e8747724257772dc3bd34092f483

Thanks!
